### ISSUE TITLE
[boston-linear-regression] Carlos Vila-San Juan

### ISSUE THREAD

**Issue by carlosvilasanjuan** [ironhack-datalabs/datamad1020#510]
https://github.com/carlosvilasanjuan/boston_linear_regression.git

**Comment by adrianacoca** [comment]
Buenos días Carlos!

La verdad es que tu ejercicio de boston linear regression está muy bien!! 🙌  
Veo que has probado haciendo un shuffle de los datos y esto está muy bien pensado pero cuando utilizas `train_test_split`, divide los datos de manera aleatoria (no coge los últimos) por lo tanto podrías haber ejecutado este código una segunda vez para comprobar los resultados. 
Más adelante, podrías haber probado diferentes modelos (como por ejemplo alguno de statsmodels) o haber cambiado el tamaño del test y comparar el rmse para cada uno de los cambios. 

Buen trabajo! 🚀 

![descarga (10)](https://user-images.githubusercontent.com/65229646/100325271-ad285b80-2fc8-11eb-8e25-85af1904ea52.jpeg)
